app/build.gradle

```diff
@@ -165,12 +165,18 @@ android {
 configurations {
     all {
         resolutionStrategy {
+            force "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
             force "com.android.support:support-annotations:${support_lib_version}"
         }
     }
 }
+kapt {
```

Comment:
if i remember correctly this generate stub style is a bit old isn't it?
ref: https://blog.jetbrains.com/kotlin/2016/12/kotlin-1-0-6-is-here/